File: src/main/java/ai/devchat/common/DevChatPathUtil.java
package ai.devchat.common;

public class DevChatPathUtil {
    public static String getWorkPath() {
//        return PathManager.getPluginsPath() + "/devchat";
//        return System.getProperty("user.home") + "/.chat";

        // TODO: change this to the .chat after testing
        return System.getProperty("user.home") + "/.chat-intellij";
    }

    public static String getDevchatBinPath() {
        return getWorkPath() + "/mamba" + "/envs/devchat/bin/devchat";
    }
}

File: src/main/java/ai/devchat/common/DevChatPathUtil.kt
package ai.devchat.common

object DevChatPathUtil {
    val workPath: String
        get() =//        return PathManager.getPluginsPath() + "/devchat";
//        return System.getProperty("user.home") + "/.chat";

            // TODO: change this to the .chat after testing
            System.getProperty("user.home") + "/.chat-intellij"
    val devchatBinPath: String
        get() = workPath + "/mamba" + "/envs/devchat/bin/devchat"
}
